module-1/lab-data_cleaning/your-code/main.ipynb

```diff
@@ -9,10 +9,12 @@
   },
```

Comment:
1. We should program always in english, this includes also the name of the variables. 
2. When you need to print the output of a cell involving a dataframe, It's better to use the method head(). This way you guarantee both you and the people you share with a better comprehension of the file you are working on. 
3. It's a best practice to make comments on what your cell code is doing. 
4. Step 9: You are expected to answer the question, How many missing values do you have in your merged dataframe? On which columns? This means that you need to get df.info() or similar methods to get the data that answer the question. Don't use a loop to get this, it's time execution and machine consuming.
5. When you check for outliers you should think about on which explanatory variables it makes sense (numeric variables) and exclude the ones that don't need to be checked. In this case, excluding id columns. 
6. I would expect some kind of analysis of the outliers found. Bonus It's not only coding but also practising analytical thinking and being able to explain both the rational and the final output.